platform/ethereum/domain.go

```diff
@@ -1,14 +1,31 @@
 package ethereum
 
 import (
-	CoinType "github.com/trustwallet/blockatlas/coin"
+	"github.com/minio/minio-go/pkg/set"
```

Comment:
remove it, just use 
switch case to check strings

Reply:
Did it as you wish.  Drawbacks though: repeated code, more error prone, bad for testability and coverage.